_posts/en/newsletters/2023-03-08-newsletter.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 'Bitcoin Optech Newsletter #241'
+permalink: /en/newsletters/2023/03/08/
+name: 2023-03-08-newsletter
+slug: 2023-03-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Alternative design for OP_VAULT:** Greg Sanders [posted][sanders
+  vault] to the Bitcoin-Dev mailing list an alternative design for
+  providing the features of the `OP_VAULT`/`OP_UNVAULT` proposal (see
+  [Newsletter #234][news234 vault]).  His alternative would add three
+  opcodes instead of two.  To provide an example:
+
+    - *Alice deposits funds in a vault* by paying a [taproot][topic
+      taproot] tree which contains at least two [tapscripts][topic
+      tapscript], one with her unvault tapscript and one with her freeze
+      tapscript, e.g. `tr(key,{trigger,freeze})`.
+
+      - The *trigger tapscript* contains her less-trusted authorization
```

Comment:
Im still green on this proposal, but above we say "two tapscripts, one with her **unvault** tapscript and one with her **freeze** tapscript" but here we say "trigger" and "freeze" tapscripts

Reply:
Good catch, thanks!  I rephrased several places to be more consistent, which I think actually simplified things and made them more clear.  Thanks!